src/NLog.Web.AspNetCore/README.md

```diff
@@ -11,8 +11,11 @@ Supported platforms:
 - For ASP.NET Core 2, .NET Standard 2.0 and .NET 4.6.1+
 
 Useful Links
-- [https://github.com/NLog/NLog.Web/releases](https://github.com/NLog/NLog.Web/releases)
-- [https://github.com/NLog/NLog/wiki/Tutorial](https://github.com/NLog/NLog/wiki/Tutorial)
-- [https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-6](https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-6)
-- [https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore)
-- [https://github.com/NLog/NLog/wiki/Logging-troubleshooting](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
+- [Change Log](https://github.com/NLog/NLog.Web/releases)
+- [Tutorial](https://github.com/NLog/NLog/wiki/Tutorial)
```

Comment:
Think this tutorial is mostly for console-applications, and should not be used as starting point for using NLog with AspNetCore-application.

Reply:
Completed